chromium_src/chrome/browser/ui/webui/tab_search/tab_search.mojom

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+module tab_search.mojom;
```

Comment:
This is optional but I'd be tempted to add a new mojom interface, rather than extending the `tab_search` one. This isn't really related to tab_search.

If it makes it much easier in terms of patches/chromium_src overrides I'm on board though

Reply:
I think it makes overrides easier, also, the PageHandler interface in chromium's tab_search.mojom has methods they need from their UI actions too.

Reply:
easy as